### PR TITLE
packages oracle-linux-8: add a support for Mroonga with Oracle Linux 8

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

**Comment by komainu8** [comment]
@kou Could you review this PR?

File: packages/mysql-community-8.0-mroonga/yum/oracle-linux-8/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ENV \
+  SCL=gcc-toolset-11
+
+RUN \
+  quiet=$([ "${DEBUG}" = "yes" ] || echo "--quiet") && \
+  dnf -y module disable mysql && \
+  dnf update -y ${quiet} && \
+  dnf install -y ${quiet} \
+    'dnf-command(builddep)' \
+    'dnf-command(download)' \
```
Comment (kou) [inline packages/mysql-community-8.0-mroonga/yum/oracle-linux-8/Dockerfile:15]: Could you sort this list in alphabetical order?

File: packages/mysql-community-8.0-mroonga/yum/oracle-linux-8/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    groonga-normalizer-mysql-devel \
+    intltool \
+    libcurl-devel \
+    libtool \
+    make \
+    pkgconfig \
+    rpm-build \
+    wget \
+    which && \
+  dnf install -y ${quiet} https://repo.mysql.com/mysql80-community-release-el8-3.noarch.rpm && \
```
Comment (kou) [inline packages/mysql-community-8.0-mroonga/yum/oracle-linux-8/Dockerfile:34]: Why do we need to install this multiple times?

Comment (kou) [reply packages/mysql-community-8.0-mroonga/yum/oracle-linux-8/Dockerfile:34]: Ah, https://repo.mysql.com/mysql-community-minimal-release-el8-1.noarch.rpm isn't the same package.

File: packages/mysql-community-8.0-mroonga/Rakefile [hunk 3]
```diff
@@ -21,6 +21,7 @@ class MySQLCommunity80MroongaPackageTask < MroongaPackageTask
   def yum_targets_default
     [
       "almalinux-8",
+      "oracle-linux-8",
       "centos-7",
```
Comment (kou) [inline packages/mysql-community-8.0-mroonga/Rakefile:25]: Could you sort this list in alphabetical order?

**Comment by kou** [comment]
Did you confirm that the current AlmaLinux 8 packages don't work on Oracle Linux 8?

**Comment by komainu8** [comment]
I close this PR.
Because the current AlmaLinux 8 packages work on Oracle Linux 8.

**Pull request closed without merging**